distribution/options.html

```diff
@@ -64,6 +64,20 @@
 				<button id="clear-cache">Clear cache</button>
 			</label>
 		</p>
+		<p>
+			<strong>Find a feature</strong>
+		</p>
+		<p>
+			This process will help you identify what Refined GitHub feature is making changes or causing issues on GitHub.
```

Comment:
Should we already here tell the user to first disable Refined GitHub to make sure the issue is with Refined GitHub?

Reply:
I had proposed it this way at the beginning but I’m not particularly interested in that, the user can just toggle the extension if they feel this situation is likely. 

If not, they can just follow the procedure as a regular bisect, and that’s just easier to reason and code for us.

Reply:
But isn't it worth trying this before wasting time on the bisect?

Reply:
Do you personally see yourself using this wizard for that step?

> But isn't it worth trying this before wasting time on the bisect?

Yes and no, fully disabling the extension is more effective than this anyway since we have non-disableable CSS features and code that runs outside `init`, e.g. https://github.com/sindresorhus/refined-github/issues/3336

I know I personally would probably disable it entirely in the first place if I'm unsure whether a bug is caused by RG, e.g. https://github.com/sindresorhus/refined-github/issues/3055#issuecomment-623152965